packages/kit/src/core/build/index.js

```diff
@@ -392,6 +385,18 @@ async function build_server(
 			const get_hooks = hooks => ({
 				getSession: hooks.getSession || (() => ({})),
 				handle: hooks.handle || (({ request, resolve }) => resolve(request)),
+				handleError: ((args) => {
```

Comment:
nit: `args` is so much like `arguments` that I got a little confused at first: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments

I might call it `opts` instead